### PR TITLE
fix consensus related comments

### PR DESCRIPTION
**Pull request by NashBC**
### Description

add a description of your changes here...

### Rationale

tell us why we need these changes...

### Example

add an example CLI or API response...

### Changes

Notable changes: 
* add each change in a bullet point here
* ...

### CODE REVIEW

File: consensus/parlia/parlia.go [hunk 1]
```diff
@@ -399,40 +406,48 @@ func (p *Parlia) verifyVoteAttestation(chain consensus.ChainHeaderReader, header
 	if attestation == nil {
 		return nil
 	}
+	if attestation.Data == nil {
+		return fmt.Errorf("invalid attestation, vote data is nil")
```
Comment (calmbeing) [inline consensus/parlia/parlia.go:410]: Better to have more useful information in the log here, like voteDate information

File: core/types/vote.go [hunk 2]
```diff
@@ -12,35 +12,41 @@ import (
...
 	BLSSignatureLength = 96
+
+	MaxAttestationExtraLength = 256
 )
 
 type BLSPublicKey [BLSPublicKeyLength]byte
 type BLSSignature [BLSSignatureLength]byte
 type ValidatorsBitSet uint64
 
+// VoteData represents the vote data that validators vote for fast finality.
```
Comment (calmbeing) [inline core/types/vote.go:23]: "voteData represents vote data"... emm, sounds the words duplicated. Maybe we can make it more sense, like VoteData represents brief blockInfo inside the vote, SourceNumber, hash, etc.

File: consensus/parlia/parlia.go (hunk 1 above)
Comment (NashBC) [reply consensus/parlia/parlia.go:410]: vote data is nil, no more info?

File: core/types/vote.go (hunk 2 above)
Comment (NashBC) [reply core/types/vote.go]: Sorry, I don't think in the comment we should use SouceNumber these variables, they are not readable, and I comment on each field in the VoteData.

**Pull request merged**